modules/log6x-log-collector-http-server.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-log-collector-http-server_{context}"]
+= Configuring the collector to receive audit logs as an HTTP server
+
+You can configure your log collector to listen for HTTP connections to only receive audit logs by specifying `http` as a receiver input in the `ClusterLogForwarder` custom resource (CR).
+
+:feature-name: HTTP receiver input
+include::snippets/logging-http-sys-input-support.adoc[]
+
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {oc-first}.
+* You have installed the {clo}.
+* You have created a `ClusterLogForwarder` CR.
+
+.Procedure
+
+. Modify the `ClusterLogForwarder` CR to add configuration for the `http` receiver input:
++
+--
+.Example `ClusterLogForwarder` CR
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+# ...
+spec:
+  inputs:
+  - name: http-receiver # <1>
+    type: receiver
+    receiver:
+      type: http # <2>
+      port: 8443 # <3>
+      http:
+        format: kubeAPIAudit # <4>
+  pipelines: # <5>
+    - name: http-pipeline
+      inputRefs:
+        - http-receiver
+# ...
+----
+<1> Specify a name for your input receiver.
+<2> Specify the input receiver type as `http`.
+<3> Optional: Specify the port that the input receiver listens on. This must be a value between `1024` and `65535`. The default value is `8443` if this is not specified.
+<4> Currently, only the `kube-apiserver` webhook format is supported for `http` input receivers.
+<5> Configure a pipeline for your input receiver.
+--
+
+. Apply the changes to the `ClusterLogForwarder` CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
```

Comment:
Can you add the expected result as below:
Once the CLF is created. The collector listens on the service <clf-name>-<input[].name>.   you can send logs to this https server.

```
oc get svc
NAME                      TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)            AGE
collector                 ClusterIP   172.30.85.239    <none>        24231/TCP          3m6s
collector-http-receiver   ClusterIP   172.30.205.160   <none>        8443/TCP           3m6s

```

Reply:
send logs to httpserver using curl

curl --cacert <openshift-service-ca.crt> https://collector-http-receiver.<namespace>.svc:8443 -XPOST -d '{"foo":"bar"}'